.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Continuous Integration
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  ci:
+    if: ${{ github.event_name == 'push' || github.event.pull_request.merged == true }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run tests
+        run: |
+          ./init.sh
+          docker compose up -d api
```

Comment:
You shouldn't need to spin up the `api` for tests, the `e2e` tests use a `TestClient` object.

Reply:
Okay, so only the db is needed, will make the change.